content/install_maintain/installation/installing-cluster.md

```diff
@@ -635,6 +649,33 @@ listen manager
 ```
 
 ### Post Installation
+
+#### Setup Cloudiy HA cluster status reporters
+
+1. First the following information needs to be collected:
+- List of the current private ip addresses of Cloudify mangement service, or at least one of them, that list will be updated automaticly afterwards.
+- Run the follwing command on anyone of Cloudify mangement service machines for retreving specific internal communication tokens:
+  ```bash
+  cfy_manager status-reporter get-tokens
+  ```
+  This will return a list of three token named 'manager_status_reporter', 'db_status_reporter' and 'broker_status_reporter', please copy aside 'db_status_reporter' and 'broker_status_reporter' tokens.
+- Please copy to all cluster nodes, expect Cloudify mangement's service ones, the CA certificate that was specified in the config.yaml at the section 'ssl_inputs' of field 'ca_cert_path'. This is for
```

Comment:
Please copy to all cluster nodes, **except ** Cloudify **management** service ones